packages/morph/lib/attr-morph.js

```diff
@@ -6,17 +6,20 @@ function updateProperty(value) {
   this.domHelper.setPropertyStrict(this.element, this.attrName, value);
 }
 
-function updateAttribute(value) {
+function updateAttribute(value, namespace) {
```

Comment:
Suppose you have `<svg><use xml:space={{space}}></use></svg>` or similar.  The `use` element will have the `namespaceURI` for svg, which is correct.  But it means that `element.namespaceURI === svgNamespace` in the outer `if` (in your code snippet) would evaluate as true for the attribute's morph, leading it to be set with `setAttribute` rather than `setAttributeNS`.  

At any rate, the PR's newer commits handle it correctly, so it's moot.